host/README.rst

```diff
@@ -0,0 +1,28 @@
+Installing xscope_fileio host tools
+-----------------------------------
+
+To install ``xscope_fileio`` host tool, please follow the steps below:
+
+1. Make sure you have a C compiler  installed. If you are developing on Windows, we recomend using VS tools with a ``cl`` compiler.
+
+2. Open a terminal or command prompt.
+
+3. Install the xscope_fileio package. From the current directory, run the following command:
+
+.. tab:: Linux and Mac
+
+  .. code-block:: console
+
+    >> Linux and Mac
+    pip install -e ../.
+
+.. tab:: Windows
+
+  .. code-block:: console
+
+    >> Windows
+    pip install -e ../.
+    cmake -G Ninja .
```

Comment:
I wasn't aware windows needed this

Reply:
On wodnwos yes, we could add a build for windows if we assume a compiler on host side in a future version